src/commands/test/test_suite.py

```diff
@@ -2,12 +2,8 @@
 from pathlib import Path
 from typing import List
 
-from src.utils.starknet_compilation import StarknetCompiler
-
 
 @dataclass
 class TestSuite:
-    TestFunction = StarknetCompiler.AbiElement
-
     test_path: Path
-    test_functions: List["TestSuite.TestFunction"]
+    test_functions: List[dict]
```

Comment:
I think string would be enough, I don't think we are going to expand this soon

Reply:
I'll do it in a different PR.